changelogs/unreleased/fix-incorrect-print-of-msgpack-field.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/msgpack
+
+* Fix incorrect printing of the msgpack field
+  with the MP_EXT type.
```

Comment:
A user reading this change log (change logs are for users) wouldn't understand what it's about. Are there any user-visible effects (like possible crash?) If so, please mention it and, even better, open a ticket for the reference. If there isn't, better drop this change log altogether.

Reply:
Remove this changelog because it this fix is not visible to user